### DISCUSSION TITLE
VS pump keeps changing speed back and forth

### DISCUSSION THREAD

**Discussion by RRoyceus** [aqualinkd/AqualinkD#268] [General]
My pool pump is a Pentair variable speed,  I have a Jandy Panel.  I erased all the programs in the panel.  In  the web browser I set the schedule to CMD RPM, Value 2350.  The pump speed keeps constantly changing  back and forth from 2350 to 2850 then back to 2350.

Any suggestions on how to to fix it?

**Comment by sfeakes** [comment]
I have not seen this before, can you post the logging information that way I can see what's going on.
BTW, How long between the speed changes.

For logging info.
set aqualinkd.conf to debug.
set log to file
Restart aqualinkd
start pump
wait for the RPM to fluctuate
post the log file

Just a thought.  If you have a lot of devices on the RS458 bus, the control panel can loose connections to the pump, and the pump will then go into default mode (if default RPM is 2850) this *might* be what the problem is.  If you post the logs, I'll be able to see.
